src/main/resources/static/api.yml

```diff
@@ -3258,6 +3258,105 @@ paths:
           $ref: '#/components/responses/403Response'
         500:
           $ref: '#/components/responses/500Response'
+    put:
+      summary: Update information about a specific user's roles and qualifications
+      parameters:
+        - name: X-Service-Name
+          in: header
+          required: true
+          description: If given, only users for this service will be returned
+          schema:
+            $ref: '#/components/schemas/ServiceName'
+        - in: path
+          name: id
+          required: true
+          description: Id of the user
+          schema:
+            type: string
+            format: uuid
+      requestBody:
+        content:
+          'application/json':
+            schema:
+              $ref: '#/components/schemas/UserRolesAndQualifications'
+        required: true
+      responses:
+        200:
+          description: Successfully added information about user roles and qualifications
+          content:
+            'application/json':
+              schema:
+                $ref: '#/components/schemas/User'
+        401:
+          $ref: '#/components/responses/401Response'
+        403:
+          $ref: '#/components/responses/403Response'
+        500:
+          $ref: '#/components/responses/500Response'
+  /users/search/{name}:
```

Comment:
I don't think we need the `/{name}` segment here. Judging from the other methods that use params it's enough to just have them specified in the `parameters:` key

Reply:
Good catch - I've made changes to this and the Delius search endpoint. Now using the form of /users/search?name=foo

Reply:
Thanks Gareth